bin/vhost-gen

```diff
@@ -766,12 +766,19 @@ def vhost_get_server_status(config, template, verbose):
     return output
 
 
-def vhost_get_custom_section(config, verbose):
-    # type: (Dict[str, Any], int) -> str
+def vhost_get_custom_section(config, template, verbose):
+    # type: (Dict[str, Any], Dict[str, Any], int) -> str
     """Get virtual host custom directives."""
     output = to_str(config["custom"])
+    if output:
+        output += "\n"
+
+    if "custom" in template:
+        output += to_str(template["custom"])
+
     log(3, "[custom]  Retrieving cfg 'custom': {}".format(output), verbose)
```

Comment:
This was one too much. The first log statement is about what has been retrieved from the config, so this should be directory below the first `output = to_str(config["custom"])` line